src/custom_components/_all.scss

```diff
@@ -0,0 +1,11 @@
+@import "./slider2.scss";
```

Comment:
Please include files in alphabetical order unless if there's some dependency between them